hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -98,6 +98,14 @@
   ]
 
   // List of test specifications.
+  //
+  // If you are adding a test that has been generated from the `sw_tests`
```

Comment:
To be clear, this is applicable when building the image for Verilator / FPGA runs as well. Perhaps this should be added to the SW build documentation.

Reply:
Hopefully the comment in `sw/device/tests/meson.build` helps too? In reality, I wrote that comment for sw-devs that are probably not as familiar with the meson targets as you are (because most of us just build everything all the time).

Reply:
Yup, fine with me either way.